scm-ui/ui-webapp/src/components/LoginForm.tsx

```diff
@@ -50,7 +50,7 @@ const AvatarImage = styled(Image)`
   width: 128px;
   height: 128px;
   padding: 5px;
-  background: #fff;
+  background: white;
```

Comment:
Unfortunately, we cannot access the css custom properties because they are not supported by all browsers. white, however, corresponds to the value of $white.